stdlib/private/StdlibCollectionUnittest/CheckSequenceType.swift

```diff
@@ -136,6 +136,50 @@ public struct FindTest {
   }
 }
 
+public struct CollectionTypeTest {
+  public let expected: [MinimalEquatableValue]
+  public let a: [MinimalEquatableValue]
+  public let b: [MinimalEquatableValue]
+  public let loc: SourceLoc
+
+  public init(
+    expected: [Int], a: [Int], b: [Int],
+    file: String = #file, line: UInt = #line
+  ) {
+    self.expected = expected.enumerated().map {
+      return MinimalEquatableValue($1, identity: $0) 
```

Comment:
I don't think that we can expect any particular identity in the result, unless we know something about the operation.  Imagine a "union" operation.  If there are identical elements in lhs and rhs, it is not specified from which collection the element will be picked.